assets/app/views/logs/_log_formatted.html

```diff
@@ -0,0 +1,49 @@
+<div>
+  <a
+    name="logTop"
+    id="logTop"></a>
+
+  <div class="log-view">
+    <div
+      ng-show="logList.length > 40"
+      class="log-scroll log-scroll-top"
+      affix
+      offset-top="285"
+      offset-bottom="90">
+      <a ng-click="scrollBottom()">
+        Go to end of log
+      </a>
+    </div>
+    <div layout flex column class="log-view-output">
+      <div class="spinner spinner-inverse" ng-show="!ready"></div>
+      <div
+        layout row
+        class="log-line"
+        ng-repeat="logLine in logList  | filter:logSearch"
+        ng-class-odd="'odd'"
+        ng-class-even="'even'">
+        <a
+          name="line_{{::$index+1}}"
+          id="line_{{::$index+1}}"></a>
+
+        <a class="log-line-number"
+          ng-click="scrollTo('line_'+($index+1), $event)">
+          <span layout flex main-axis="end">
+          {{:: $index+1 }}
+          </span>
+        </a>
+        <span flex class="log-line-text">
+        {{::logLine.text}}
+        </span>
+      </div>
+    </div>
```

Comment:
Here is where the rendering and filtering has the potential to get a little ugly. 

The current impl works fine for up to 15,000 lines (that I've tested).  There is a delay in render at that point, but we have a loading spinner that makes this clear.  Once rendered with one way binding `{{::var}}` the page does not seem to have issues. Links to top, bottom & individual line #'s work.

 I've got an experimental branch off of this branch where I have tried a couple different angular methods, then a jQuery raw DOM string method and an _.template() method.  There does not seem to be much of a performance improvement, with enough lines each one become sluggish.  

@jwforres @spadgett @sg00dwin